docs/glossary.rst

```diff
@@ -94,6 +95,23 @@ email alert if something seems wrong. Only system admins connect
 to this server, and they may only do so using Tor.
 
 
+Onion Service
+-------------
+
+Tor onion services provide anonymous inbound connections to websites and other servers exclusively over the Tor network. For example, SecureDrop uses onion services for the *Journalist Interface* and *Source Interface* websites, as well as for adminsitrative access to the servers in SSH-over-Tor mode.
+
+Onion services can be accessed by clicking a link or pasting the onion service address into Tor Browser. For example, `sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion <https://sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion>`__ is SecureDrop's onion service address.
+
+Onion Service v2
+""""""""""""""""
+
+The third generation of onion services (v3) provides stronger cryptographic primitives (algorithms) than v2 onion services, and include redesigned protocols that guard against service information leaks on the Tor network.
+
+It is strongly recommended that only v3 onion services are used with SecureDrop 1.7.0 and forward.
```

Comment:
```suggestion
It is strongly recommended that only v3 onion services are used from SecureDrop 1.7.0 onwards.
```

Reply:
We can strengthen this recommendation now:

> Support for v2 onion services will start to be removed from SecureDrop in February 2021, and we strongly recommend [upgrading SecureDrop instances to v3 onion services](https://docs.securedrop.org/en/stable/v3_services.html) as soon as possible.